charts/dispatch/charts/function-manager/templates/config-map.yaml

```diff
@@ -12,7 +12,8 @@ data:
         "host": "{{ .Values.faas.openwhisk.host }}"
       },
       "openfaas": {
-        "gateway": "{{ .Values.faas.openfaas.gateway }}"
+        "gateway": "{{ .Values.faas.openfaas.gateway }}",
+        "func_namespace": "{{ .Values.faas.openfaas.namespace }}"
```

Comment:
Can you use camelcase?  I see we will have a small merge conflict :).

Reply:
Sure! I'll use the names from your PR :)

Reply:
No worries, I'm going to have a merge conflict anyway.  I can easily deal with it.